Lib/test/test_ast.py

```diff
@@ -91,24 +104,38 @@ def to_tuple(t):
     "if v:pass",
     # If-Elif
     "if a:\n  pass\nelif b:\n  pass",
+    # If-Else
+    "if a:\n  pass\nelse:\n  pass",
     # If-Elif-Else
     "if a:\n  pass\nelif b:\n  pass\nelse:\n  pass",
     # With
+    "with x: pass",
+    "with x, y: pass",
     "with x as y: pass",
     "with x as y, z as q: pass",
     # Raise
     "raise Exception('string')",
+    # Raise from
```

Comment:
Unless there is a reason otherwise, I would remove this and add `"raise Exception",` first.

Reply:
You mean add `raise Exception` without creating an instance of `Exception`? Yeah, seems good. Thanks!
About comment - we can remove it, and add "(from)" in the same way as in `yield (from)` case.